### PR TITLE
feat: duration field use dropdown+text in performance preference

### PR DESCRIPTION
**Pull request by PmcFizz**
Signed-off-by: pmc <pangmengchen@deepexi.com>

**Description**

This PR fixes https://github.com/layer5io/meshery/issues/1411

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@PmcFizz dang, that was fast! Before @kushthedude could even get a screenshot taken. 😉 

I love seeing someone move faster than @kushthedude. It doesn't happen all that often. 😄

**Comment by leecalcote** [comment]
@Aisuko, where'd you find this guy (@PmcFizz) anyway?! 😜

File: ui/components/MesherySettingsPerformanceComponent.js [hunk 1]
```diff
@@ -77,7 +87,7 @@ class MesherySettingsPerformanceComponent extends React.Component {
 
     if (t === '' || !(t.toLowerCase().endsWith('h')
       || t.toLowerCase().endsWith('m') || t.toLowerCase().endsWith('s')) || err || tNum <= 0) {
-      this.setState({ tError: true });
+      this.setState({ tError: 'error-autocomplete-value' });
```
Comment (kushthedude) [inline ui/components/MesherySettingsPerformanceComponent.js:90]: @PmcFizz Everything seems awesome, just wanted to know why this change?

Comment (PmcFizz) [reply ui/components/MesherySettingsPerformanceComponent.js:90]: If the duration  field test don't pass , will add a error class `error-autocomplete-value` to components Autocomplete for show  red error border。 this style of class write on `style/index.css`

Comment (kushthedude) [reply ui/components/MesherySettingsPerformanceComponent.js:90]: But that is what tError: true also does I guess

Comment (PmcFizz) [reply ui/components/MesherySettingsPerformanceComponent.js:90]: If other component maybe tError  is true what is ok. but components ` Autocomplete `  is  not. you can try。

if tError is false，the  border will not change red 。

**Review by kushthedude — APPROVED** [review]

File: ui/components/MesherySettingsPerformanceComponent.js (hunk 1 above)
Comment (kushthedude) [reply ui/components/MesherySettingsPerformanceComponent.js:90]: Yes, I was talking in respective of textfield & this is autocomplete! Thanks!

**Comment by leecalcote** [comment]
Whoo-hoo! Thanks, @PmcFizz 💪
